test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
```

Comment:
I think we could, via composition, compose multiple schedulerPerfConfigs to support scenarios where there are multiple competing node affinities in the future.  Either way i think leaving as-is for now is good for the first iteration: We can expand it further in the future, and this is a step forward from what is already there...  Is that ok with you @gmarek ?